index_config/mappings.works_indexed.2024-11-06.json

```diff
@@ -106,6 +106,9 @@
                 },
                 "id": {
                   "type": "keyword"
+                },
+                "sourceId": {
+                  "type": "keyword"
```

Comment:
Should we [use a normalizer](https://github.com/wellcomecollection/catalogue-pipeline/blob/b9ba99f0f383cbf6ba3f0d03d5a2ec6b0536d1dc/index_config/mappings.works_indexed.2024-11-06.json#L737) like we do elsewhere?

Reply:
I can't decide if these should be called `sourceIdentifier` following the pattern elsewhere, or if they are a different kind of thing and justify another name?

Reply:
Yes, good point, I added a lowercase normalizer. 

I also renamed `sourceId` to `sourceIdentifier` to be more consistent with other source ID fields.